lib/smart_proxy_openscap/arf_parser.rb

```diff
@@ -0,0 +1,109 @@
+# encoding=utf-8
+require 'openscap'
+require 'openscap/ds/arf'
+require 'openscap/xccdf/testresult'
+require 'openscap/xccdf/ruleresult'
+require 'openscap/xccdf/rule'
+require 'openscap/xccdf/fix'
+require 'openscap/xccdf/benchmark'
+require 'json'
+require 'digest'
+
+module Proxy::OpenSCAP
```

Comment:
this only works if this script is ran from smart proxy process so Proxy is already defined, if you break it to

```
module Proxy
  module OpenSCAP
    class ArfParser
```
and change all `OpenSCAP` references to `::OpenSCAP` it will work even standalone (might be useful during debugging)